src/data/resolvers/motions.ts

```diff
@@ -331,6 +333,17 @@ export const motionsResolvers = ({
         });
       }
 
+      if (
+        motionNetworkState === NetworkMotionState.Closed &&
+        !motionDomainId.eq(ROOT_DOMAIN_ID)
+      ) {
+        systemMessages.push({
+          type: ActionsPageFeedType.SystemMessage,
+          name: SystemMessagesName.MotionCanBeEscalated,
+          createdAt: motion.events[2].toNumber() * 1000,
```

Comment:
why are we multiplying by 1000?

Reply:
Because that's the format the time from the events array is, in seconds, and we need milliseconds